### DISCUSSION TITLE
Details regarding server location, data flow for data protection compliance

### DISCUSSION THREAD

**Discussion by AgniKai95** [mozilla/hubs#5631] [General Questions & Discussion]
Dear hubs-team,
 
we are looking at hosting a business event in Mozilla Hubs, preferably using the public not self hosted version, but are running into issues with our strict internal guidelines regarding data protection and GRDP compliance in general.
 
At the moment we are facing main issues:
1.	Server location outside of the EU (us-east-1)
Egressing Data flows and data procession outside of the EU usually result in a refusal from our data protection team.  Do you offer a (paid) solution for hosting a hubs cloud event using aws-infrastructure located inside of the EU (preferably Germany, eu-central-1)? Or is the only alternative available the usage of a self-hosted Hubs Cloud Enterprise environment?
2.	Data flows egressing the EU
If using aws-infrastructure in Europe (public hubs / self hosted) are there any (separate) data flows potentially egressing the EU? E.G. Mozilla-Accounts/account information mentioned in the privacy policy and necessary for moderator rights/room configuration.
3.	Do you offer a DPA?
4.	Details regarding data processing mentioned in the privacy policy
For internal approval, we need a complete and more detailed list of data processed, transmission and storage, encryption and data retention compared to the information derivable from the privacy policy. It is missing some details (highlighted) which we need for internal approval. 
See this table:
![image](https://user-images.githubusercontent.com/97593111/182398599-6379e625-d370-4739-bc9a-14853007c8db.png)

Additonally the abstract technical data from the privacy policy contains the vague term “other data”. (We receive data about the type of device you use to interact with Hubs, as well as its operating system, language, the name and version of browser, and other data needed to load and operate a room.)
What does it include apart from the data mentioned?
 
Thanks a lot

**Comment by pattersonbl2** [comment]
This is currently a known issue with hubs.  here is a link to issue https://github.com/mozilla/hubs/issues/5396#issuecomment-1132214233

**Reply by AgniKai95** [reply]
The issue linked is focusing mostly on the ability to remove one specific user´s data upon request. We are mainly looking for details regarding data flows missing in the privacy policy (table above)
